src/frontend/src/components/LogIn/index.tsx

```diff
@@ -166,6 +166,24 @@ class LogIn extends React.Component<Props, State> {
                 Forgot your password?
               </Link>
             </div>
+            <div className="tc">
+              <Link
+                to="/add-user"
+                className="link underline hover-blue"
+                href="/"
+              >
+                Go to Add User Component
```

Comment:
I would prefer "Register" or "Create a new user"

Reply:
Perhaps this is a placeholder? Please see issue #593 for details.